README.md

```diff
@@ -59,3 +63,5 @@ This is a curated list of tools and services supporting the Digital Humanities.
 * [Duke University's DH Tools catalog](https://digitalhumanities.duke.edu/tools) - This list includes tools that Duke supports and tools that have been used by Duke digital projects. Some of the tools are made specifically for DH and others that can be re-purposed quite effectively for Humanities research.
 
 * [Carolina Digital Humanities Initiative Tools Page](http://digitalhumanities.unc.edu/resources/tools/) - It provides a range of platforms, plug-ins, readings, and other items that might be of use for DH researchers.
+
+* [Taxonomy of Digital Research Activities in the Humanities (TaDiRAH)](http://tadirah.dariah.eu/vocab/) - This taxonomy has been developed for use by community-driven sites and projects that aim to structure information relevant to digital humanities and make it more easily discoverable. The taxonomy is expected to be particularly useful to endeavors aiming to collect information on digital humanities tools, methods, projects, or readings.
```

Comment:
@diegosiqueir4 I know this has been merged already and sorry for a very late review...
However, don't you think this should not be part of this chapter/list, a taxonomy compared to other registries seem a bit different.
However, teresah.dariah.eu (which also uses TaDIRAH) might fit here.

Reply:
You're absolutely right, @yoannspace . Thanks for your comment.
That also raises another point for discussion: I believe this repository has grown more than the original scope (DH tools) since it now also supports organizations, tutorials, user guides,...

What do you think about changing the scope of this repository from "Awesome DH Tools" to "Awesome DH"? This change would give us more freedom to create a more comprehensive list and be able to create sessions that could, for example, incorporate contributions like TaDiRAH.

Reply:
to avoid confusion, I will create another pull request with the discussion that originated in this topic.